plugins/python/python.py

```diff
@@ -417,7 +417,8 @@ def _on_py_command(word, word_eol, userdata):
         unload_name(name)
     elif subcmd == 'reload':
         name = ffi.string(word[3]).decode()
-        reload_name(name)
+        if not reload_name(name):
+            print("cannot find a plugin with that name")
```

Comment:
Lets keep the messages the same as the previous plugin: `Can't find a python plugin with that name`

Reply:
changed in f2133bfd011eab013823a568a6e885c21b532458